pam/integration-tests/testdata/TestSSHAuthenticate/golden/prevent_user_from_switching_username

```diff
@@ -0,0 +1,630 @@
+> ssh ${AUTHD_PAM_SSH_USER}@localhost ${AUTHD_PAM_SSH_ARGS}
+(user-integration-pre-check-prevent-user-from-switching-username@localhost)
+== Broker selection ==
+1 - local
+2 - ExampleBroker
+Select broker:
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Why are there so many empty lines in these golden files? They blow up the diff a lot and make it harder to review. Could we strip them, or is there any in use in keeping them?

Reply:
So, we can make the golden shorter in some cases, but not in this one unfortunately.

Because as [you can see](https://github.com/ubuntu/authd/blob/69699df13506a308e3b99efe0d199cfeedda93ba/pam/integration-tests/testdata/TestSSHAuthenticate/golden/prevent_user_from_switching_username) the last page of the golden file is completely full.

Now, the fact is that when using VHS you define the size of your terminal window, and that view is not scrollable, so in order to see everything that a script will add in such view during its runtime, you can only make it longer.

This different from the case of bubbletea based tapes, where bubble tea at each redraw clears the view completely, but this is not something that a native PAM interface or SSH can do for us.

Reply:
It was somehow explained in the golden files before, see https://github.com/ubuntu/authd//commit/ef60945ee89e1cb458e8fc17f60ec89e2ee4f733 and https://github.com/charmbracelet/vhs/issues/404

Reply:
I understand that VHS produces those empty lines, but why can't we strip them from the output (for example in [`sanitizeGoldenFile`](https://github.com/ubuntu/authd/blob/main/pam/integration-tests/ssh_test.go#L267-L276)) before comparing/storing the golden files?

Reply:
Yes, we could... I will probably craft another PR for that though (and then I'll rebase this on that, ok?)

Reply:
Sounds good!

Reply:
=> https://github.com/ubuntu/authd/pull/598